Makefile

```diff
@@ -13,12 +13,12 @@ GOPATH_DEFAULT := $(PWD)/.go
 export GOPATH ?= $(GOPATH_DEFAULT)
 TESTARGS_DEFAULT := "-v"
 export TESTARGS ?= $(TESTARGS_DEFAULT)
-PKG := $(shell awk  '/^package: / { print $$2 }' glide.yaml)
+PKG := $(shell awk  -F "\"" '/^ignored = / { print $$2 }' Gopkg.toml)
```

Comment:
I think this is going to return the entire list from the toml file:
    `ignored = ["k8s.io/cloud-provider-openstack"]`

When we need just:
    `k8s.io/cloud-provider-openstack`

Unless my awk is super rusty, which it probably is :)